pkg/controllers/user/monitoring/metricExpression.go

```diff
@@ -2422,134 +2425,8 @@ spec:
-  legendFormat: CPU usage([[container_name]])
-  description: container cpu usage seconds sum rate
----
-kind: MonitorMetric
-apiVersion: management.cattle.io/v3
-metadata:
-  name: container-cpu-system-seconds-sum-rate-details
-  labels:
-    app: metric-expression
-    component: container
-    details: "true"
-    graph: cpu-usage
-    level: project
-    source: rancher-monitoring
-spec:
-  expression: sum(rate(container_cpu_system_seconds_total{namespace=~"$namespace",pod_name=~"$podName",
-    container_name=~"$containerName"}[5m]))
-  legendFormat: CPU system seconds([[container_name]])
-  description: container cpu system seconds sum rate
----
-kind: MonitorMetric
-apiVersion: management.cattle.io/v3
-metadata:
-  name: container-cpu-user-seconds-sum-rate-details
-  labels:
-    app: metric-expression
-    component: container
-    details: "true"
-    graph: cpu-usage
-    level: project
-    source: rancher-monitoring
-spec:
-  expression: sum(rate(container_cpu_user_seconds_total{namespace=~"$namespace",pod_name=~"$podName",
-    container_name=~"$containerName"}[5m]))
-  legendFormat: CPU user seconds([[container_name]])
-  description: container cpu user seconds sum rate
----
-kind: MonitorMetric
-apiVersion: management.cattle.io/v3
-metadata:
-  name: container-memory-usage-bytes-sum-details
-  labels:
-    app: metric-expression
-    component: container
-    details: "true"
-    level: project
-    graph: memory-usage-bytes-sum
-    source: rancher-monitoring
-spec:
-  expression: sum(container_memory_working_set_bytes{name!~"POD", namespace=~"$namespace",pod_name=~"$podName",
-    container_name=~"$containerName"})
-  legendFormat: Memory usage([[container_name]])
-  description: container memory usage bytes sum
----
-kind: MonitorMetric
-apiVersion: management.cattle.io/v3
-metadata:
-  name: container-disk-io-writes-bytes-sum-rate-details
-  labels:
-    app: metric-expression
-    component: container
-    details: "true"
-    graph: disk-io
-    level: project
-    metric: disk-io-writes-bytes-sum-rate
-    source: rancher-monitoring
-spec:
-  expression: sum(rate(container_fs_writes_bytes_total{namespace=~"$namespace",pod_name=~"$podName",
-    container_name=~"$containerName"}[5m]))
-  legendFormat: Write([[container_name]])
-  description: container disk io writes bytes sum rate
----
-kind: MonitorMetric
-apiVersion: management.cattle.io/v3
-metadata:
-  name: container-disk-io-reads-bytes-sum-rate-details
-  labels:
-    app: metric-expression
-    component: container
-    details: "true"
-    graph: disk-io
-    level: project
-    metric: disk-io-reads-bytes-sum-rate
-    source: rancher-monitoring
-spec:
-  expression: sum(rate(container_fs_reads_bytes_total{namespace=~"$namespace",pod_name=~"$podName",
-    container_name=~"$containerName"}[5m]))
-  legendFormat: Read([[container_name]])
-  description: container disk io reads bytes sum rate
----
-#### Container Metrics ####
-
```

Comment:
move the deleted container metric code to another commit

Reply:
Rollbacked